index.js

```diff
@@ -31,22 +35,47 @@
 // // => 11
 // console.log("%c----------", "color: red") 
 
+// function drinkWater(current_thirst_level) {
+//   if (current_thirst_level -= 1){
+//     return "Man I sure am thirsty"
+//   }
+//   else{
+//     return "Ahh that hits the spot"
+//   }
+// }
+
+function drinkWater(currentThirstLevel) {
+  console.log("Man, I sure am thirsty")
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel - 1
+}
+console.log(drinkWater(12))
```

Comment:
Good! 💯 No need for conditionals in this function because this function should always subtract 1 from the currentThirstLevel and should always console log those two statements